nextcloud_update.sh

```diff
@@ -778,13 +778,33 @@ then
     print_text_in_color "$ICyan" "Restoring the status of apps. This can take a while..."
     for app in "${!APPSTORAGE[@]}"
     do
-        if ! is_app_installed "$app"
+        if [ -n "${APPSTORAGE[$app]}" ]
         then
-            nextcloud_occ_no_check app:install "$app"
-        fi
-        if [ -n "${APPSTORAGE[$app]}" ] && [ "${APPSTORAGE[$app]}" != "yes" ] && is_app_enabled "$app"
-        then
-            nextcloud_occ_no_check config:app:set "$app" enabled --value="${APPSTORAGE[$app]}"
```

Comment:
We cannot get rid of this because it would break https://github.com/nextcloud/server/issues/4538 again